marker_api/src/sem/ty.rs

```diff
@@ -13,6 +15,8 @@ pub use sequence_ty::*;
 pub use trait_ty::*;
 pub use user_ty::*;
 
+use crate::common::DriverTyId;
```

Comment:
Somehow the uses landed in different spots: one before `pub use`s and one after. Let's put them all after `pub use`s because I think this is where Rust analyzer would put them by default.